src/db/migrations.rs

```diff
@@ -61,6 +61,7 @@ fn all_migrations() -> Vec<Box<dyn Migration>> {
     create table repos_jiras (
         repo_id integer not null,
         jira varchar not null,
+        channel varchar not null,
```

Comment:
taking the easy road here again on the db migrations since this isn't deployed anywhere yet.